assignments/lambda-classes.js

```diff
@@ -1 +1,208 @@
+   `sprintChallenge` similar to PRAssignment but logs out `student.name has begun sprint challenge on {subject}`
+ */
+
+class Student extends Person {
+    constructor (studentinfo){
+        super(studentinfo);
+        this.previousBackground = studentinfo.previousBackground;
+        this.className = studentinfo.className;
+        this.favSubjects = studentinfo.favSubjects;
+    }
+
+    listsSubjects() {
+        console.log(...this.favSubjects);
+    }
+
+    PRAssignment(student, subject){
+        console.log(`${student.name} has submitted a PR for ${subject}`)
+    }
+
+    sprintChallenge(student, subject){
+        console.log(`${student.name} has begun sprint challenge on ${subject}`)
+    }
+}
+
+//Project Mananger
+/* Now that we have instructors and students, we'd be nowhere without our PM's
+ ProjectManagers are extensions of Instructors
+ ProjectManagers have the following uniqe props:
+   `gradClassName`: i.e. CS1
+   `favInstructor`: i.e. Sean
+ ProjectManangers have the following Methods:
+   `standUp` a method that takes in a slack channel and logs `{name} announces to {channel}, @channel standy times!​​​​​
+   `debugsCode` a method that takes in a student object and a subject and logs out `{name} debugs {student.name}'s code on {subject}` */
+
+   class ProjectManagers extends Instructor {
+    constructor (pm){
+        super (pm);
+        this.gradClassName = pm.gradClassName;
+        this.favInstructor = pm.favInstructor;
+    }
+
+    standUp(pm, channel) {
+        console.log(`${pm.name} announces to ${channel}, @channel standy times!`)
+    }
+
+    debugsCode(pm, student, subject) {
+        console.log(`${pm.name} debugs ${student.name}'s code on ${subject}`)
+    }
+}
+
+  //Instructors
+
+  const Jackie = new Instructor ({
+    name: 'Jackie',
+    age: 30, 
+    location: 'Bali',
+    gender: 'Female',
+    specialty: "JavaScript",
+    favLanguage: "Ruby",
+    catchPhrase: "Jackie Jackie Jackie Jackie!"
+});
+
+const Tati = new Instructor ({
+    name: 'Tati',
+    age: 36, 
+    location:'Los Angelos',
+    gender: 'Female',
+    specialty: "React",
+    favLanguage: "Javascript",
+    catchPhrase: "She sells seashells by the seashore"
+});
+
+const Nikki = new Instructor ({
+    name: 'Nikki',
+    age:  24, 
+    location: 'Germany',
+    gender: 'Female',
+    specialty: "Django",
+    favLanguage: "PhP",
+    catchPhrase: "C'est une belle journee"
+});
+
+   //PMs
+
+   const Gabe = new ProjectManagers ({
+    name: 'Gabe',
+    age: 23, 
+    location: 'Cali',
+    gender: 'Male',
+    gradClassName: "CS12",
+    favInstructor: "Comet"
+});
+
+const Lola = new ProjectManagers ({
+    name: 'Lola',
+    age: 24, 
+    location: 'Virginia',
+    gender: 'Female',
+    gradClassName: "CS14",
+    favInstructor: "Dasher"
+});
+
+const Ronald = new ProjectManagers ({
+    name = 'Ronald',
+    age = 22, 
+    location = 'West Coast',
+    gender = 'Male',
```

Comment:
You accidentally used `=` (_assignment operator_) instead of `:` here. Inside an object literal, you use `:`, not `=`, to define key/value pairs. 

Of course, I think you know this, but just made a little mistake. Running the code here should have clued you in of the syntax error here, which underscores the importance of running the code and console logging things.